DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: streamMetabolizer
 Type: Package
 Title: Models for Estimating Aquatic Photosynthesis and Respiration
-Version: 0.9.29
-Date: 2016-12-28
+Version: 0.9.30
```

Comment:
You don't also want to specify a minimum version of `rstan` to go along w/ this change?